recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-nfs-no-coproc.cfg

```diff
@@ -0,0 +1,34 @@
+# =====================================================================
+# Example PV Linux guest configuration
+# =====================================================================
+#
+# This is a fairly minimal example of what is required for a
+# Paravirtualised Linux guest. For a more complete guide see xl.cfg(5)
+
+seclabel='system_u:system_r:domU_t'
+
+# Guest name
+name = "DomU"
+
+# Kernel image to boot
+kernel = "/boot/domu/Image"
+
+device_tree = "/xen/domu.dtb"
```

Comment:
yes, this was my intention. but strictly speaking this has no relation to "no-coproc", so @aanisov comment is valid